--- clex-3.1.8/src/control.c.orig	2003-02-27 22:37:52.000000000 +0100
+++ clex-3.1.8/src/control.c	2004-03-28 18:36:12.302539160 +0200
@@ -17,7 +17,9 @@
 
 #include <sys/types.h>		/* clex.h */
 #include <ctype.h>			/* iscntrl() */
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/curses.h>
+#elif defined(HAVE_NCURSES_H)
 # include <ncurses.h>		/* KEY_xxx */
 #else
 # include <curses.h>
--- clex-3.1.8/src/inout.c.orig	2003-03-02 18:16:34.000000000 +0100
+++ clex-3.1.8/src/inout.c	2004-03-28 18:36:46.509338936 +0200
@@ -19,7 +19,10 @@
 #include <ctype.h>		/* iscntrl() */
 #include <stdarg.h>		/* va_list */
 #include <string.h>		/* strcpy() */
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/curses.h>
+# include <ncurses/term.h>
+#elif defined(HAVE_NCURSES_H)
 # include <ncurses.h>	/* initscr() */
 #else
 # include <curses.h>
--- clex-3.1.9/configure.in.orig	2004-04-05 19:42:34.000000000 +0000
+++ clex-3.1.9/configure.in	2004-08-21 18:15:50.417702432 +0000
@@ -19,7 +19,7 @@
 AC_HEADER_MAJOR
 AC_HEADER_SYS_WAIT
 AC_HEADER_TIME
-AC_CHECK_HEADERS([locale.h ncurses.h sys/time.h term.h ncurses/term.h])
+AC_CHECK_HEADERS([locale.h ncurses/ncurses.h sys/time.h term.h ncurses/term.h])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_HEADER_STAT
