--- clex-3.1.8/src/control.c.orig	2003-02-27 22:37:52.000000000 +0100
+++ clex-3.1.8/src/control.c	2004-03-28 18:36:12.302539160 +0200
@@ -17,7 +17,9 @@
 
 #include <sys/types.h>		/* clex.h */
 #include <ctype.h>			/* iscntrl() */
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/curses.h>
+#elif defined(HAVE_NCURSES_H)
 # include <ncurses.h>		/* KEY_xxx */
 #else
 # include <curses.h>
--- clex-3.1.8/src/inout.c.orig	2003-03-02 18:16:34.000000000 +0100
+++ clex-3.1.8/src/inout.c	2004-03-28 18:36:46.509338936 +0200
@@ -19,7 +19,10 @@
 #include <ctype.h>		/* iscntrl() */
 #include <stdarg.h>		/* va_list */
 #include <string.h>		/* strcpy() */
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/curses.h>
+# include <ncurses/term.h>
+#elif defined(HAVE_NCURSES_H)
 # include <ncurses.h>	/* initscr() */
 #else
 # include <curses.h>
--- clex-3.1.8/configure.in.orig	2003-02-24 11:28:12.000000000 +0100
+++ clex-3.1.8/configure.in	2004-03-28 18:37:06.524296200 +0200
@@ -14,7 +14,7 @@
 AC_HEADER_MAJOR
 AC_HEADER_SYS_WAIT
 AC_HEADER_TIME
-AC_CHECK_HEADERS(ncurses.h fcntl.h unistd.h term.h ncurses/term.h)
+AC_CHECK_HEADERS(ncurses.h ncurses/ncurses.h fcntl.h unistd.h term.h ncurses/term.h)
 AC_DECL_SYS_SIGLIST
 AC_C_CONST
 AC_TYPE_UID_T
